tools/config/__init__.py

```diff
@@ -1058,9 +1062,31 @@ def validate_config(self):
 
         Arguments: None
         """
+
         params, _ = self.get_config_data()
+        err_msg = ""
+
+        for name, param in sorted(params.items()):
+            min      = param.value_min
+            max      = param.value_max
+            accepted = param.accepted_values
+            value    = param.value
+
+            if (value < min or (value > max if max is not None else False)):
```

Comment:
Don't you also need to do the `if is not None` check for `min` here?

```suggestion
            if (value < min if min is not None else False) or (value > max if max is not None else False):
```

Reply:
I can to make it look more consistent, but it's sort of surprisingly not necessary. None is less than any other datatype in Python (unless later versions of Python 3 have changed that).